var/spack/repos/builtin/packages/libexif/package.py

```diff
@@ -8,14 +8,21 @@
 class Libexif(AutotoolsPackage, SourceforgePackage):
     """A library to parse an EXIF file and read the data from those tags"""
 
-    homepage = "https://sourceforge.net/projects/libexif/"
-    sourceforge_mirror_path = "libexif/libexif-0.6.21.tar.bz2"
+    homepage = "https://libexif.github.io/"
+    url = "https://github.com/libexif/libexif/releases/download/v0.6.24/libexif-0.6.24.tar.bz2"
 
     maintainers("TheQueasle")
 
-    license("LGPL-2.0-or-later")
+    license("LGPL-2.1-or-later", checked_by="wdconinc")
 
+    version("0.6.24", sha256="d47564c433b733d83b6704c70477e0a4067811d184ec565258ac563d8223f6ae")
     version("0.6.21", sha256="16cdaeb62eb3e6dfab2435f7d7bccd2f37438d21c5218ec4e58efa9157d4d41a")
 
-    depends_on("c", type="build")  # generated
+    depends_on("c", type="build")
     depends_on("glib")
+
+    def url_for_version(self, version):
+        if version <= Version("0.6.21"):
```

Comment:
```suggestion
        if self.spec.satisfies("@:0.6.21"):
```

Quick suggestion for using `spec.satisfies()` here instead of creating a new version object.